go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/argoproj/argo-cd/v2
 
-go 1.19
+go 1.21
```

Comment:
@robinlieb is bumping this a necessity? I think this defines the _lowest_ supported version, which may be able to stay at 1.19 for now.

Reply:
I think 1.19 is past EOLhttps://endoflife.date/go